pom.xml

```diff
@@ -158,64 +158,66 @@
         <module>cloud/aws-common</module>
         <module>cloud/gcp-common</module>
         <!-- Core extensions -->
-        <module>extensions-core/kubernetes-extensions</module>
-        <module>extensions-core/avro-extensions</module>
-        <module>extensions-core/azure-extensions</module>
+        <!-- <module>extensions-core/kubernetes-extensions</module> -->
+        <!-- <module>extensions-core/avro-extensions</module> -->
+        <!-- <module>extensions-core/azure-extensions</module> -->
         <module>extensions-core/datasketches</module>
-        <module>extensions-core/druid-bloom-filter</module>
-        <module>extensions-core/druid-kerberos</module>
+        <!-- <module>extensions-core/druid-bloom-filter</module> -->
+        <!-- <module>extensions-core/druid-kerberos</module> -->
         <module>extensions-core/druid-pac4j</module>
-        <module>extensions-core/hdfs-storage</module>
-        <module>extensions-core/histogram</module>
-        <module>extensions-core/stats</module>
-        <module>extensions-core/kafka-extraction-namespace</module>
+        <!-- <module>extensions-core/hdfs-storage</module> -->
+        <!-- <module>extensions-core/histogram</module> -->
+        <!-- <module>extensions-core/stats</module> -->
+        <!-- <module>extensions-core/kafka-extraction-namespace</module> -->
         <module>extensions-core/kafka-indexing-service</module>
-        <module>extensions-core/kinesis-indexing-service</module>
-        <module>extensions-core/multi-stage-query</module>
-        <module>extensions-core/mysql-metadata-storage</module>
-        <module>extensions-core/orc-extensions</module>
-        <module>extensions-core/parquet-extensions</module>
+        <!-- <module>extensions-core/kinesis-indexing-service</module> -->
+        <!-- <module>extensions-core/multi-stage-query</module> -->
+        <!-- <module>extensions-core/mysql-metadata-storage</module> -->
+        <!-- <module>extensions-core/orc-extensions</module> -->
+        <!-- <module>extensions-core/parquet-extensions</module> -->
         <module>extensions-core/postgresql-metadata-storage</module>
-        <module>extensions-core/protobuf-extensions</module>
-        <module>extensions-core/lookups-cached-global</module>
-        <module>extensions-core/lookups-cached-single</module>
-        <module>extensions-core/ec2-extensions</module>
+        <!-- <module>extensions-core/protobuf-extensions</module> -->
```

Comment:
the protobuf extensions are useful to have to debug data streams, we already depend on the same dependencies in our other modules.